tsup/dict.ts

```diff
@@ -0,0 +1,4 @@
+type KeyValuePair<K extends string | number | symbol, V> = [K, V];
```

Comment:
Could this be:
```suggestion
type KeyValuePair<K extends PropertyKey, V> = [K, V];
```

since `PropertyKey` = `string | number | symbol`

Reply:
I had the dict function from previous webpack js file, and I just lazily used claude to type it for me, without thinking :)